jobs/JGLOBAL_PREP_OBS_AERO

```diff
@@ -39,7 +39,7 @@ fi
 ##########################################
 # Remove the Temporary working directory
 ##########################################
-cd "${DATAROOT}" || exit
+cd "${DATAROOT}" || true
```

Comment:
Not familiar with this syntax.  What does this do?

Reply:
If `DATAROOT` does not exist, then `cd "${DATAROOT}"` will return a non-zero status, but we do not want that to cause an error, so `|| true` is triggered, which always returns a `0` status.  `|| exit` will do something similar, and I think in practice that would be fine, but elsewhere in the code `|| true` is used, so I just wanted to keep things the same.

Reply:
Thank you @DavidHuber-NOAA for explaining what this does.  Good to learn new things.